example/k8s-aliyun-ubuntu/k8s-node-rusali.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: archon.kubeup.com/v1
```

Comment:
change the file name to `k8s-node` and add another step in readme about creating nodes